frontend/src/pages/About.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+<div class="bg-white dark:bg-gray-900">
```

Comment:
Why these colors, the parent tag in App.vue already has these classes: `bg-background dark:bg-background-dark text-foreground dark:text-foreground-dark`

Reply:
I cleaned all color with scheme classes.